code/game/machinery/tcomms/core.dm

```diff
@@ -0,0 +1,263 @@
+#define UI_TAB_CONFIG "CONFIG"
+#define UI_TAB_LINKS "LINKS"
+
+/**
+  * # Telecommunications Core
+  *
+  * The core of the entire telecomms operation
+  *
+  * This thing basically handles the main broadcasting of the data, as well as NTTC configs
+  * The relays dont do any actual processing, they are just objects which can bring tcomms to another zlevel
+  */
+/obj/machinery/tcomms/core
+	name = "Telecommunications Core"
+	desc = "A large rack full of communications equipment. Looks important."
+	icon_state = "core"
+	/// The NTTC config for this device
+	var/datum/nttc_configuration/nttc = new()
```

Comment:
Shouldn't we delete this when the core is destroyed?

Reply:
Added to the destroy